packages/rpc-core/src/rpc-methods/common.ts

```diff
@@ -19,3 +19,5 @@ export type RpcResponse<TValue> = Readonly<{
     }>;
     value: TValue;
 }>;
+
+export type Lamports = U64UnsafeBeyond2Pow53Minus1 & { readonly __lamports: unique symbol };
```

Comment:
Mmmm… I think this is more like…

```suggestion
export type LamportsUnsafeBeyond2Pow53Minus1 = bigint & { readonly __lamports: unique symbol };
```

Reply:
As you wish! In that case, do you also want to do `SlotUnsafeBeyond2Pow53Minus1`?

Reply:
I asked how long it would take to get there and @willhickey [told me](https://discord.com/channels/428295358100013066/439194979856809985/1075222932936216647) ~114M years, so I think I decided against it. 

<img width="543" alt="image" src="https://user-images.githubusercontent.com/13243/234405160-f224f67d-3066-4e47-891f-d03112fedda7.png">

Reply:
Not opposed to it, for completeness, though.

Reply:
ah gotcha, that definitely makes sense